### PR TITLE
Account cancellation: Export archive

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-e8fab9520d/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-5058555db2/how-to/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-8813a10a4f/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-3439a9736f/how-to/). :bow:

**Comment by stephamon** [comment]
@DonNateR here are my suggestions. Let me know if you have any questions!

[Export.an.archive.before.cancelling.a.Cloud.Office.account_SF.docx](https://github.com/rackerlabs/rackspace-how-to/files/1398493/Export.an.archive.before.cancelling.a.Cloud.Office.account_SF.docx)
[Export.an.archive.before.cancelling.a.Cloud.Office.account_SF.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1398494/Export.an.archive.before.cancelling.a.Cloud.Office.account_SF.pdf)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-32e9a30901/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-060a1ff8c8/how-to/). :bow:

File: content/rackspace-email-archiving/export-an-archive-before-cancelling-a-Cloud-Office-account.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+The pricing for a bulk export is a one-time fee and varies by amount of data as well as data format.
+
+|Export Format| Price per Gigabyte(GB)|
+|---|---|
+|EML| $6 or &#65505;4 per GB|
+|PST| $9 or &#65505;6 per GB|
+
+
+**Note:** Support cannot to provide a quote for the price of requested bulk exports. This is because the total size of the export cannot be determined until it is completed and fully exported. The archive size displayed in your archive portal is the size of your article in compressed form. After your archive is decompressed, it is larger than the size
```
Comment (stephamon) [inline content/rackspace-email-archiving/export-an-archive-before-cancelling-a-Cloud-Office-account.md]: Looks like the rest of the note got cut off here.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-086667335a/how-to/).

**Comment by stephamon** [comment]
👍

**Pull request merged**